docs/developer/branching-and-workflow.md

```diff
@@ -10,9 +10,26 @@ way ephemeral developer branches will remain outside the main repository.
 Below is an example of following the merge and fork process. Developer Alice:
 
 - Forks `github.com/nginxinc/nginx-kubernetes-gateway` → `github.com/<alice-user-id>/nginx-kubernetes-gateway`
+- Adds upstream:
+  ```shell
+  git remote add upstream git@github.com:nginxinc/nginx-kubernetes-gateway.git
+  ```
+  Alice now has the follow remotes configured:
+  ```shell
+  git remote -v
```

Comment:
Should probably add `$`, or indent the output just to show separation between command and output.